app/controllers/spree/admin/base_controller.rb

```diff
@@ -12,6 +12,12 @@ class BaseController < Spree::BaseController
       before_filter :set_locale
       before_filter :warn_invalid_order_cycles, if: :html_request?
 
+      # Temporary measure to help debugging strong_parameters
+      rescue_from ActiveModel::ForbiddenAttributesError, with: :print_params
+      def print_params
+        raise ActiveModel::ForbiddenAttributesError, params.to_s
+      end
```

Comment:
Do we really need this one? spree_admin_base inherits from spree_base (in spree core) that inherits from appcontroller above

Reply:
Removed.